src/MicrosoftTeams.ts

```diff
@@ -21,14 +21,26 @@ namespace microsoftTeams {
 
   const version = "1.2";
 
-  const validOrigins = [
-    "https://teams.microsoft.com",
-    "https://teams.microsoft.us",
-    "https://int.teams.microsoft.com",
-    "https://devspaces.skype.com",
-    "https://ssauth.skype.com",
+  const validHostRegEx = new RegExp(
```

Comment:
NIT: Consider making this a little easier to extend by creating the regex dynamically in code out of an array